test/org/opensolaris/opengrok/configuration/messages/StatsMessageTest.java

```diff
@@ -159,8 +159,8 @@ public void testInvalidReload() {
 
```

Comment:
Actually I meant that you provide a non-existing file, like "/no/file/exists/g65g65d4g65v16516" and then the test will raise an exception. [see this comment](https://github.com/OpenGrok/OpenGrok/pull/1440/files/45776d90dfb381e32043317d9a853d06b3cac0b4#r102430071) which might solve the situation while not changing anything in the test